tex/chapters/08-Conclusion.tex

```diff
@@ -0,0 +1,51 @@
+In Chapter \ref{cha:literature} an overview of literature related to Prisoner's Dilemma was presented, with particular emphasis on the IPD and several computer tournaments that have be held.
+This was followed by a description of several common strategies that have been used throughout the report.
+Also, an outline of literature relating to Finite State Machines and how they are relevant within IPD is given.
+This is follow by a description of how Axelrod's work is currently being reproduced by an open-source community.
+Finally, an introduction to Fingerprinting (a technique used for identifying similar strategies) is given.
+
+In Chapter \ref{cha:theory} many definitions were presented culminating in a rigorous definition for a Fingerprint.
+It was then shown that any strategy can be represented as a Finite State Machine, allowing the work of Ashlock to be extended to include any strategy to be used as a probe.
+Finally, an example of how to construct an Analytical Fingerprint was provided.
+
+In Chapter \ref{cha:implementation}, the process of implementing Fingerprint in Axelrod-Python is explained, including several algorithms equivalent to definitions from Chapter \ref{cha:theory}.
+Comparisons with work produced in previous literature are made to ensure that results produced are correct.
+
+In Chapter \ref{cha:results} several fingerprints were examined in detail.
+This included discussions of individual fingerprints, a comparison of fingerprints for a strategy with a varying parameter, and also the importance of probe selection.
+
+In Chapter \ref{cha:machinelearning} a new approach to identifying similar strategies using Machine Learning was given.
+The process of training and assessing the model was described in detail, followed by some preliminary results of the models performance.
+Finally, in Chapter \ref{cha:applying-model}, the model was used to identify equivalent strategies.
+First, a level of the model's sensitivity was obtained using a variety of MemoryOne strategies from Axelrod-Python.
+This was followed by the model being applied to all strategies within Axelrod-Python; producing network graphs of strategies the model considered identical.
+
+
+
+\section{The Future}
+
+Due to time constraints, there were many interesting topics that could not be investigated fully.
+Future work could include examining more Fingerprint plots, in a similar to Chapter \ref{cha:results}.
+Explaining the appearance of Fingerprints for a given strategy and probe is not trivial, especially for strategies with variable parameter.
+In addition, using more probes to produce Fingerprints to study the effect they have.
+
+In Chapters \ref{cha:machinelearning} and \ref{cha:applying-model}, only one probe is used to produce the data the model is trained on.
+The probe was selected to be in line with previous literature and it is unlikely that it is the best choice.
+The possibility of using multiple probes to train the model was also not considered.
```

Comment:
The real future work would be to identify a prob best identifies similar behaviour and thanks to your work of chapter ... we're not constrained by FSM strategies and could make use of some highly trained strategies currently in the library.